src/main/java/io/appform/dropwizard/discovery/bundle/ServiceDiscoveryConfiguration.java

```diff
@@ -78,21 +86,27 @@ public class ServiceDiscoveryConfiguration {
 
     private Set<String> tags;
 
+    @JsonIgnore
+    public Set<String> getZookeeperHosts() {
+        return Arrays.stream(zookeeper.split(ZOOKEEPER_HOST_DELIMITER))
+                .map(zkHostPort -> zkHostPort.split(HOST_PORT_DELIMITER)[0])
```

Comment:
Can you check if a path can be provided in this conn string? if yes, those need to be removed before parsing

Reply:
Splitting on "/" and taking the hostname now